scripts/ingests/utils.py

```diff
@@ -65,6 +65,70 @@ def load_simpledb(db_file, RECREATE_DB=True):
     return db
 
 
+def find_in_simbad(sources, desig_prefix, source_id_index = None, verbose = False):
+    """
+    Function to extract source designations from SIMBAD
+
+    Parameters
+    ----------
+    sources
+        Sources names to search for in SIMBAD
```

Comment:
This looks like a table, is that right?

Reply:
Honestly, I am having a hard time knowing when I’m passing a list vs a table. Advice on how to be more explicit about this would be helpful

Reply:
That's entirely up to you based on how you write the function. I thought it looked like a table since further below you refer to `sources['source']` and, to the best of my knowledge, you can not refer to values in a list by a string.

To make it clearer here you can do one of several things:
1. Explicitly write out the type after sources (eg, `sources : astropy.table.Table`)
2. Update the description to make it clearer (eg, `Sources Astropy Table to search for in SIMBAD`)
3. Use the type-hinting that Will has been recommending, so in the function parameters you say sources : astropy.table.Table
In my own code, I tend to use a combination of 1 and 2 to make things a little clearer.